### PR TITLE
Aqcu 1539

### PR DESCRIPTION
**Pull request by kmschoep-usgs**

### CODE REVIEW

File: R/utils-validation.R [hunk 1]
```diff
@@ -28,6 +28,7 @@ isEmpty <- function(val){
 #' @seealso \code{\link{isEmpty}}
 #'
 isEmptyOrBlank <- function(val = NULL, listObjects = NULL, objectName = NULL){
+	thingval <- val
```
Comment (mhines-usgs) [inline R/utils-validation.R]: did you mean to leave this in?

File: R/utils-validation.R [hunk 2]
```diff
@@ -163,6 +164,9 @@ validateFetchedData <- function(data, name, requiredFields, stopNull=TRUE, stopM
   }
 
   #Check for valid but empty data
+	thing <- class(data)
```
Comment (mhines-usgs) [inline R/utils-validation.R]: and these?

File: tests/testthat/test-correctionsataglance.R [hunk 3]
```diff
@@ -822,16 +821,16 @@ test_that("getLaneLabelData properly calculates the label positon data for each
...
+        "levelDescription": "Approved",
         "comment": "Approval changed to Approved by gwilson.",
         "dateApplied": "2016-05-19T16:26:58.2093803Z",
         "startTime": "2017-01-01T12:12:13",
         "endTime": "2017-02-01T12:12:13"
       }
     ]
   }')
-
+browser()
```
Comment (mhines-usgs) [inline tests/testthat/test-correctionsataglance.R]: debugging?

File: R/utils-validation.R (hunk 1 above)
Comment (kmschoep-usgs) [reply R/utils-validation.R]: oops, no.

**Comment by zmoore-usgs** [comment]
Coverage looks good now 👍 `coverage/coveralls — Coverage remained the same at 97.408% `

**Review by zmoore-usgs — COMMENTED** [review]
Check out my comments here about the correction type: https://github.com/USGS-CIDA/aqcu-corr-report/pull/2#event-1833640367

**Comment by mhines-usgs** [comment]
not sure if it matters much, but there are other changes to names of the data in the json output that weren't applied to the test data, it may not break anything as far as I can tell (?) but here they are

- approvals.dateApplied -> approvals.dateAppliedUtc
- (new) approvals.user
- qualifiers.appliedBy -> qualifiers.user

**Comment by kmschoep-usgs** [comment]
thanks, I will update those in the code and update the test files.

**Comment by mhines-usgs** [comment]
@zmoore-usgs didn't we need both type and dominant type for TSS? I'm having a hard time remembering why but maybe it doesn't need to be that way with CORR.

**Comment by zmoore-usgs** [comment]
Going by this commit it seems like we changed everything to dominant type?

https://github.com/USGS-R/repgen/commit/e961889c0bf7d2c1fdf4fb54346e8b6e7d090698

**Review by mhines-usgs — APPROVED** [review]

**Pull request merged**